mailbox/cassandra/src/main/java/org/apache/james/mailbox/cassandra/modules/CassandraAclModule.java

```diff
@@ -33,14 +33,15 @@
 import org.apache.james.mailbox.cassandra.table.CassandraACLTable;
 
 import com.datastax.driver.core.schemabuilder.SchemaBuilder;
+import com.google.common.collect.ImmutableList;
 
 public class CassandraAclModule implements CassandraModule {
 
     private final List<CassandraTable> tables;
     private final List<CassandraType> types;
 
     public CassandraAclModule() {
-        tables = Collections.singletonList(
```

Comment:
*SingletonList* is already immutable:
```java
    /**
     * Returns an immutable list containing only the specified object.
     * The returned list is serializable.
     *
     * @param  <T> the class of the objects in the list
     * @param o the sole object to be stored in the returned list.
     * @return an immutable list containing only the specified object.
     * @since 1.3
     */
    public static <T> List<T> singletonList(T o) {
        return new SingletonList<>(o);
    }
```